pkg/server/combined_statement_stats.go

```diff
@@ -676,18 +690,32 @@ FROM (SELECT fingerprint_id,
 			ie,
 			// The statement activity table has aggregated metadata.
 			`
-SELECT *
-FROM (SELECT fingerprint_id,
-             array_agg(distinct transaction_fingerprint_id),
-             app_name,
-             max(aggregated_ts)                                                AS aggregated_ts,
-             crdb_internal.merge_aggregated_stmt_metadata(array_agg(metadata)) AS metadata,
-             crdb_internal.merge_statement_stats(array_agg(statistics))        AS statistics
-      FROM %s %s
-      GROUP BY
-          fingerprint_id,
-          app_name) %s
-%s`,
+WITH Aggregated AS (
+    SELECT fingerprint_id,
+      array_agg(distinct transaction_fingerprint_id) AS txn_fingerprints,
+      app_name,
+      max(aggregated_ts) AS aggregated_ts,
+      crdb_internal.merge_stats_metadata(array_agg(metadata)) AS merged_metadata,
```

Comment:
If this query is for the activity table, it needs to use `crdb_internal.merge_aggregated_stmt_metadata` because the metadata column in the activity table is already aggregated from the full stmts system table.